platform/saivpp/vpplib/SwitchStateBase.h

```diff
@@ -1016,6 +1018,12 @@ namespace saivpp
 		_In_ sai_object_id_t tbl_oid,
 		_In_ bool is_add);
 
+            sai_status_t tbl_hw_ports_map_delete(
```

Comment:
Please add some documents to explain these new functions

Reply:
Added